src/Monolog/Processor/IntrospectionProcessor.php

```diff
@@ -55,6 +55,14 @@ public function __invoke(array $record)
             return $record;
         }
 
+        /*
+        * http://php.net/manual/en/function.debug-backtrace.php
+        * As of 5.3.6, DEBUG_BACKTRACE_IGNORE_ARGS option was added.
+        * Any version less than 5.3.6 must use the DEBUG_BACKTRACE_IGNORE_ARGS constant value '2'.
+        */
+        if (!defined('DEBUG_BACKTRACE_IGNORE_ARGS')) {
+            define('DEBUG_BACKTRACE_IGNORE_ARGS', 2);   
```

Comment:
I would not define internal constants. Doing so may break other libraries which use this constant for version/feature detection.